projects/saas-ui/src/app/on-boarding/adapters/validate-lead-adapter.service.ts

```diff
@@ -0,0 +1,15 @@
+// tslint:disable:rule no-any
+import {Injectable} from '@angular/core';
+import {ApiModule, IAdapter} from '@project-lib/core/api';
+
+@Injectable({
+  providedIn: ApiModule,
+})
+export class ValidateLeadAdapter implements IAdapter<any> {
+  adaptToModel(resp: any): any {
+    return resp.body;
+  }
+  adaptFromModel(data: any): any {
+    return data;
+  }
```

Comment:
If there is nothing specific and we are using any types only 
then why not use AnyAdapter